### DISCUSSION TITLE
Rechnungen, Sollbuchungen und Buchungszuordnungen

### DISCUSSION THREAD

**Discussion by tolot27** [openjverein/jverein#492] [Q&A]
Servus!

Irgendwie komme ich noch nicht so ganz mit der Rechnungslegung und der Zuordnung der eingegangenen Zahlungen klar. Bisher habe ich Zusatzbeträge erfasst und per Lastschrift in einer kompakten Buchung eingezogen. Da hat alles gut funktioniert. Auch einzelne Beträge lassen sich wunderbar zuordnen.
Nun habe ich jedoch die Situation, dass mehrere Zusatzbeträge eines Mitglieds per Überweisung zu zahlen sind. Da gibt es ja keine "kompakten" Überweisungen, sondern es wird für jeden Zusatzbetrag eine Sollbuchung erzeugt. Mehrere davon kann man dann einer Rechnung zuweisen.
Allerdings finde ich keine Möglichkeit, wie dann der Zahlungseingang der Rechnung zugewiesen werden kann. Zur Auswahl stehen da zwar alle Sollbuchungen. Es lässt sich jedoch immer nur eine Sollbuchung der Überweisung zuordnen.
Hier sollte es möglich sein, einen Zahlungseingang auch einer Rechnung zuordnen zu können.
Außerdem wäre es dann auch gut, wenn man in der Rechnungsansicht sehen würde, ob sie ausgeglichen ist, oder nicht. Momentan sieht man das nur bei den einzelnen Sollbuchungen.

**Comment by mbmueller** [comment]
> Allerdings finde ich keine Möglichkeit, wie dann der Zahlungseingang der Rechnung zugewiesen werden kann. Zur Auswahl stehen da zwar alle Sollbuchungen. Es lässt sich jedoch immer nur eine Sollbuchung der Überweisung zuordnen.

Dafür musst du dann eine Splitbuchung erzeugen.

**Comment by JohannMaierhofer** [comment]
> > Allerdings finde ich keine Möglichkeit, wie dann der Zahlungseingang der Rechnung zugewiesen werden kann. Zur Auswahl stehen da zwar alle Sollbuchungen. Es lässt sich jedoch immer nur eine Sollbuchung der Überweisung zuordnen.
> 
> Dafür musst du dann eine Splitbuchung erzeugen.

So würde ich das auch sehen.
